_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Agile Integrations Workshop - Citizen Integrator
+menu:
+  main:
+    parent: workshops
+    pre: fa fa-play-circle
+---
+## Description
+
+The Agile Integration Workshop teaches the development, testing, and deployment of cloud-native and connected solutions.  You will walk through an API Development and Agile Integration process leveraging OpenShift, Apicurio, Microcks, Fuse, and 3scale for a sample application.
```

Comment:
Line 10:(Second sentence) In this workshop, you will walk through API Development and an Agile Integration processes that leverage OpenShift, Apicurio.....